datahub-web-react/src/app/identity/user/UserListItem.tsx

```diff
@@ -75,6 +101,34 @@ export default function UserListItem({ user, canManageUserCredentials, onDelete
     const userStatus = user.status; // Support case where the user status is undefined.
     const userStatusToolTip = userStatus && getUserStatusToolTip(userStatus);
     const userStatusColor = userStatus && getUserStatusColor(userStatus);
+    const rolesMap: Map<string, DataHubRole> = new Map();
+    roles.forEach((role) => {
+        rolesMap.set(role.urn, role);
+    });
+
+    const mapRoleIcon = (roleName) => {
```

Comment:
Yeah this is duplicate static function - let's extract!

Reply:
Refactored!